site/styles/_meta-detail.scss

```diff
@@ -66,6 +66,7 @@
     right: 0px;
     margin-left: auto;
     margin-right: auto;
+    height: 100%;
```

Comment:
Why do you need this with the `body` being `100%` `flex`?

Reply:
This was due to an issue where the height of the background was set larger than the height of the body at some resolutions. 
 
![image](https://user-images.githubusercontent.com/41402426/100798785-fb39d500-3488-11eb-8b28-28656dbfc788.png)

I did `height: 100%` so that it is only ever as tall as its parent (.page) until it reaches `max-height: 900px`. This works in IE11.